site/docs/tasks/custom-taxonomy.md

```diff
@@ -51,3 +51,123 @@ helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait --set-file taxo
 ```
 The `--set-file` flag will pass in your custom `taxonomy.json` file to use for taxonomy validation in Fybrik.
 If this flag is not provided, Fybrik will use the default `taxonomy.json` file with no layers compiled into it. 
+
+
+After installing the fybrik helm chart, it is possible to upgrade Fybrik with an updated custom taxonomy file (`taxonomy.json`) with the following command:
+
+```bash
+helm upgrade fybrik fybrik-charts/fybrik -n fybrik-system --wait --set-file taxonomyOverride=taxonomy.json
+```
+
+
+## Examples of changing taxonomy
+
+- Example 1: Add new intent for FybrikApplication
```

Comment:
I would suggest adding a description of what is being done in this example.  ...
In this example we show how to update the application taxonomy. We show that when a FybrikApplication yaml containing a marketing intent is submitted, it's validation fails because initially the application taxonomy does not include `marketing`.  We then describe how to add `marketing` to the taxonomy, enabling the validation to pass when we re-submit the FybrikApplication yaml